tdns/dnshandler.go

```
@@ -185,6 +197,23 @@ func createHandler(conf *Config) func(w dns.ResponseWriter, r *dns.Msg) {
 				qname = strings.ToLower(qname)
 			}
 
+			if zd.Error && zd.ErrorType != RefreshError {
+				log.Printf("DnsHandler: Qname is %q, which is a known zone, but it is in %s error state: %s",
```

golangci-lint (1.62.2)

printf: log.Printf format %s has arg zd.ErrorType of wrong type github.com/johanix/tdns/tdns.ErrorType

(govet)